src/Symfony/Component/Mailer/Transport/Smtp/SmtpTransport.php

```diff
@@ -156,7 +156,7 @@ protected function parseMessageId(string $mtaResult): string
     {
         $regexps = [
             '/250 Ok (?P<id>[0-9a-f-]+)\r?$/mis',
-            '/250 Ok:? queued as (?P<id>[A-Z0-9]+)\r?$/mis',
+            '/250 (Ok:?|Message) queued as (?P<id>[a-fA-Z0-9]+)\r?$/mis',
```

Comment:
There is already `i` (camelcase) flag
```suggestion
            '/250 (Ok:?|Message) queued as (?P<id>[A-Z0-9]+)\r?$/mis',
```